phpunit.xml

```diff
@@ -13,6 +13,9 @@
       <directory>tests/bundle/</directory>
       <exclude>tests/bundle/Functional</exclude>
     </testsuite>
+    <testsuite name="Ibexa REST Contracts">
```

Comment:
it might be unexpected for someone to see it here, as it's not a common practice, so maybe we should be more specific?
```suggestion
    <testsuite name="Ibexa REST unit tests for Contracts">
```

Reply:
Unit tests testsuite isn't described as unit tests either, so I would not expand the name unless it's done in both.

Reply:
> Unit tests testsuite isn't described as unit tests either, so I would not expand the name unless it's done in both.

So now we have
* Ibexa REST Bundle
* Ibexa REST Contracts
* Ibexa REST
* Ibexa REST Server Tests

The inconsistency is already there. The point here is that contracts are expected in our ecosystem to be tested as integration tests, while here we have unique exception to that rule. As a side comment I would add that `--testsuite` is PHPUnit option and making them named with spaces makes things difficult for command line usage.

Reply:
I've moved the contract tests into "Ibexa REST" testsuite instead, as an additional directory.

4f7b62df

I think it's way better this way (we might even consider adding bundle tests there too, since we rarely if ever test them separately). WDYT?